src/frontends/pytorch/src/op/argmax_argmin.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/squeeze.hpp"
+#include "openvino/op/topk.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+namespace {
+OutputVector create_argmax_argmin_op(const NodeContext& context, TopKMode mode) {
+    num_inputs_check(context, 3, 3);
+    auto input = context.get_input(0);
+    bool keep_dims = false;
+    auto k = context.mark_node(v0::Constant::create(element::i32, Shape{}, {1}));
+    Output<Node> indices;
+    if (!context.input_is_none(2)) {
+        keep_dims = context.const_input<bool>(2);
+    }
+    if (!context.input_is_none(1)) {
+        auto axis = context.const_input<int64_t>(1);
+        auto topk = context.mark_node(std::make_shared<v3::TopK>(input, k, axis, mode, TopKSortType::NONE));
```

Comment:
Please pay attention to the case with repeating elements. If the same max element is repeating several times. what index should be reported? You need to properly select `sort` and `stable` attributes for this. Also please add explicit tests case for this for which you need to use special data generator

Reply:
first index should be reported, my parameters aligned with behavior for PyTorch onnx export, test for repeated values added